platform/o.n.core/test/qa-functional/src/org/netbeans/core/validation/ValidateLayerConsistencyTest.java

```diff
@@ -905,7 +905,11 @@ private boolean skipFile(FileObject fo) {
 
     public void testKeymapOverrides() throws Exception { // #170677
         List<String> warnings = new ArrayList<String>();
-        FileObject[] keymaps = FileUtil.getConfigFile("Keymaps").getChildren();
+        FileObject keymapRoot = FileUtil.getConfigFile("Keymaps");
```

Comment:
Turned out to be a problem when `commit-validation` was run on `clustrer.config=platform` (no keymaps there)